scripts/delete_files_with_no_metadata/delete_files_with_no_metadata.py

```diff
@@ -0,0 +1,107 @@
+import json
+import time
+
+import requests
+import subprocess
+
+DRY_RUN = False
+TOKEN = 'insert-token-without-bearer-prefix'
+SUBMISSION_URL = 'https://api.ingest.archive.data.humancellatlas.org/submissionEnvelopes/insert-submission-id'
+SUBMISSION_UUID = 'insert-submission-uuid'
+FILES_TO_DELETE_FILENAME = f'{SUBMISSION_UUID}_files_to_delete.json'
+
+
+def run(command: str, input: str = None, verbose: bool = True):
+    parsed_command = command.split(' ')
+    proc = subprocess.Popen(parsed_command,
+                            stdin=subprocess.PIPE,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            )
+    if input:
+        stdout, stderr = proc.communicate(input.encode())
+    else:
+        stdout, stderr = proc.communicate()
+
+    if verbose:
+        print('$ ' + command)
+        print(stdout.decode())
+        print(stderr.decode())
+
+    return proc.returncode, stdout.decode(), stderr.decode()
+
+
+def remove_file(s3_file_path):
+    if DRY_RUN:
+        command = f'echo "Removing file {s3_file_path}"'
+    else:
+        command = f'aws s3 rm {s3_file_path} --profile embl-ebi'
```

Comment:
Does choosing AWS CLI instead of the python library make sense?

Reply:
I'm just more familiar with aws cli. I just quickly made this script to fix submissions with extra files as a workaround. Pushing here so that any dev in ops could reuse it. Ideally, we should support this as a feature in Ingest.